pkg/controller/installation/windows/calico_windows_upgrader.go

```diff
@@ -60,6 +61,8 @@ type calicoWindowsUpgrader struct {
 	syncPeriod        time.Duration
 	installChan       chan *operatorv1.InstallationSpec
 	install           *operatorv1.InstallationSpec
+	isDegraded        bool
```

Comment:
Don't these 2 need to be locked with a mutex?

Reply:
Thanks, yes `isDegraded` needs a mutex - it's being read from the core controller goroutine and set by the calicoWindowsUpgrader goroutine.